contracts/staking/validatorShare/ValidatorShare.sol

```diff
@@ -36,7 +36,9 @@ contract ValidatorShare is IValidatorShare {
         uint256 combinedStakePower = validatorStake.add(activeAmount); // validator + delegation stake power
         uint256 _rewards = combinedStakePower.mul(_reward).div(_stakePower);
 
-        uint256 _validatorRewards = validatorStake.mul(_rewards).div(combinedStakePower);
+        uint256 _validatorRewards = validatorStake.mul(_rewards).div(
```

Comment:
Can you remove this line, got leaked after my merge :)

Reply:
sure checking

Reply:
removed.